### PR TITLE
Docs: adding missing image toolbars.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: adding missing image toolbars. Part of https://github.com/ckeditor/ckeditor5/issues/15284

---

### Additional information

_Link demo - the code is pushed but still throws an error about missing component._

_Editor placeholder demo was not updated due to technical problem._

### CODE REVIEW

File: packages/ckeditor5-style/docs/_snippets/features/styles.js
Comment (gorzelinski) [inline packages/ckeditor5-style/docs/_snippets/features/styles.js:1]: A similar error with the balloon: `The target DOM node for the feature tour balloon does not exist`.

**Review by gorzelinski — CHANGES_REQUESTED** [review]

File: docs/_snippets/features/placeholder.js
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/placeholder.js:1]: Already reset that one.

File: docs/_snippets/features/wproofreader.js
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/features/wproofreader.js:1]: This is a preexisting error and it's known.

File: packages/ckeditor5-style/docs/_snippets/features/styles.js
Comment (godai78) [reply packages/ckeditor5-style/docs/_snippets/features/styles.js:1]: This is a preexisting error and it's known.

File: docs/_snippets/installation/getting-and-setting-data/build-autosave-source.js
[the remark this answers is not in the document]
Comment (godai78) [reply docs/_snippets/installation/getting-and-setting-data/build-autosave-source.js:1]: Getting and setting data is part of the Installation section, this PR only covers the Features section. The call to this source file is utilised by the Autosave feature guide.

**Pull request merged**